src/components/layout/Nav.jsx

```diff
@@ -38,6 +39,8 @@ export default function Nav({
   const setUserProfile = useStatic(state => state.setUserProfile)
   const feedback = useStatic(state => state.feedback)
   const setFeedback = useStatic(state => state.setFeedback)
+  const resetfilters = useStatic(state => state.resetfilters)
+  const setResetFilters = useStatic(state => state.setResetFilters)
```

Comment:
Leave `resetFilters` here, but I would move `const setResetFilters = useStatic(state => state.setResetFilters)` into the `ResetFilters.jsx` file itself. Since you're not using it in `Nav.jsx`, other than passing it through to the dialog, may as well move it to the dialog itself, particularly since then you will have a "clean" component, without any props passed through.

Reply:
@kamieniarz 

* move `const setResetFilters = useStatic(state => state.setResetFilters)` line to `ResetFilters.jsx`
* replace `export default function ResetFilters({ setResetFilters })` with `export default function ResetFilters()` in `ResetFilters.jsx`
* replace `<ResetFilters setResetFilters={setResetFilters} />` with `<ResetFilters />` in `Nav.js`

Haven't checked myself but should be everything you need.